quipucords/quipucords/tests_feature_flags.py

```diff
@@ -0,0 +1,154 @@
+"""Test the FeatureFlag class and helper function."""
+
+import os
+import unittest
+from unittest import mock
+
+import pytest
+
+from .featureflag import FeatureFlag, get_feature_flags_from_env
+
+
+@pytest.mark.parametrize(
+    'env_name,env_value,feature_name,feature_value',
+    (('QPC_FEATURE_TEST', '0', 'TEST', False),
+     ('QPC_FEATURE_TEST', '1', 'TEST', True)
+     )
+)
+def test_get_feature_flags_from_env(env_name, env_value, feature_name,
+                                    feature_value):
+    """Tests if function retrieves new variables set in .env."""
+    with mock.patch.dict(os.environ, {env_name: env_value}):
+        dict_with_test_flags = get_feature_flags_from_env()
+        assert feature_name in dict_with_test_flags
+        assert dict_with_test_flags[feature_name] is feature_value
+
+
+def test_if_returns_default_value_if_another_env_set():
+    """Tests if function also returns variables in default dict."""
+    with mock.patch.dict(os.environ, ({'QPC_FEATURE_TEST': '0'})):
+        dict_with_test_flags = get_feature_flags_from_env()
+        assert 'OVERALL_STATUS' in dict_with_test_flags
+        assert 'TEST' in dict_with_test_flags
+        assert dict_with_test_flags['OVERALL_STATUS'] is False
+        assert dict_with_test_flags['TEST'] is False
+
+
+@pytest.mark.parametrize(
+    'env_name,env_value,feature_name,feature_value',
+    (('QPC_FEATURE_OVERALL_STATUS', '1', 'OVERALL_STATUS', True),
+     ('QPC_FEATURE_TEST', '0', 'TEST', False)
+     )
+)
+def test_if_value_for_env_default_list_gets_updated(
+        env_name, env_value, feature_name, feature_value):
```

Comment:
[OPTIONAL] This is an overall comment on the style here. I'm not a huge fan on discussing code style but this still bothers me on how parenthesis are closed. Here's my suggestion:

```suggestion
@pytest.mark.parametrize(
    "env_name,env_value,feature_name,feature_value",
    (
        ("QPC_FEATURE_OVERALL_STATUS", "1", "OVERALL_STATUS", True),
        ("QPC_FEATURE_TEST", "0", "TEST", False),
    ),
)
def test_if_value_for_env_default_list_gets_updated(
    env_name, env_value, feature_name, feature_value
):
```

In a future PR I'll add black to the project so we won't need to discuss this :stuck_out_tongue: